src/assignments/candidateQuiz.rst

```diff
@@ -0,0 +1,180 @@
+.. _candidateQuiz:
+
+Assignment #1: Candidate Testing
+================================
+
+OK, staff, we received many applications for our astronaut training program,
+and we need to do an initial evaluation of the candidates.  Management needs
+you to create a quick quiz to help select the best candidates.
+
+.. note::
+
+   The requirements below are what your END assignment will look like.
+   This assignment is broken down so you can complete small pieces as you go. You need to move sequentially starting at Part 1.
+   Please read the WHOLE assignment page before starting.
+
+Requirements
+------------
+
+#. Ask the candidate to enter their name
+#. Use a loop to ask five questions, one at a time, to the candidate
+#. Collect the candidate's answers
+#. Check those answers for accuracy (case insensitive equality check)
+#. Calculate the candidate's overall percentage
+#. Determine if the candidate did well enough to enter our program (need >= 80%
+   to pass)
+#. Display the results.
+
+Take It Step by Step
+--------------------
+
+When starting any project, it's best to approach it as a series of smaller,
+testable parts. The goal is to get simple parts working first and then expand
+the code in a systematic way. The following is NOT the only way to complete
+this assignment, but it provides a framework for thinking through the project.
+
+Create a replit.com project to start building your project.  Make sure "Candidate Quiz" is part of the name.
+Use the textbook for directions.  
```

Comment:
This is probably a placeholder for something, no?
I'm thinking we should probably provide the students with a starter code repl.it that already contains a main method. what do you think? It could be a good idea down the line to have them create a main method in a blank replit. But for the first graded assignment, I think it's wise to be more generous and provide it

Reply:
not a problem.